source/_integrations/tplink_eap.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: TP-Link EAP
+description: Instructions on how to use a TP-Link EAP access point as a device tracker.
+logo: tp-link.jpg
```

Comment:
This should not be used for new documentation, use our Brands repository instead (and remove this line).

Reply:
Line has been removed.